tests/bluetooth/host/keys/bt_keys_get_addr/src/main.c

```diff
@@ -100,6 +114,16 @@ void test_no_equal_references(void)
 	}
 }
 
+/* Setup test variables */
+static void every_test_setup(void *f)
```

Comment:
`test_case_setup`? What you have is also fine.

Reply:
looks like this macro will allow us to get rid of the `LISTIFY` hack in the buf.c UTs.

Reply:
Ok, I was just trying to distinguish test cases setup functions from test suite setup ones.

The names used by the test framework `ztest_suite_setup_t` `ztest_suite_before_t` `ztest_suite_after_t` `ztest_suite_teardown_t` are not very descriptive to me.

Reply:
It's actually better defined in `ztest.rst`.
- `setup`: run once at the beginning of the suite
- `before`: run at the beginning of each test case
- `after`: run at the end of each test case
- `teardown`: run once at the end of the suite